content/en/docs/collector/configuration.md

```diff
@@ -247,28 +247,31 @@ receivers:
   zipkin:
 ```
 
+> For detailed receiver configuration, see the
+> [receiver README](https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/README.md).
+
 ## Processors <img width="35" class="img-initial" src="/img/logos/32x32/Processors.svg"> {#processors}
 
-Processors are run on data between being received and being exported. Processors
-are optional though
-[some are recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors).
+Processors take the data collected by receivers and modify or transform it
+before sending it to the exporters. Data processing happens according to rules
+or settings defined for each processor, which might include filtering, dropping,
+renaming, or recalculating telemetry, among other operations.
 
-The `processors:` section is how processors are configured. Processors may come
-with default settings, but many require configuration. Any configuration for a
-processor must be done in this section. Configuration parameters specified for
-which the processor provides a default configuration are overridden.
+Processors are optional, although some
+[are recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors).
 
-> Configuring a processor does not enable it. Processors are enabled via
-> pipelines within the [service](#service) section.
+You can configure processors using the `processors` section of the Collector
+configuration file. Some processors may require security settings. Any setting
```

Comment:
The line about requiring security settings could use a link to either an example or be removed. In theory any components may require security settings in order to function :)

Reply:
Always in for _less_ docs when it makes sense. :) Removing.